src/olympia/reviewers/templates/reviewers/review.html

```diff
@@ -223,21 +223,23 @@ <h3 id="history">
   <div class="more-actions-inner">
     <ul>
       {% if not addon.is_deleted %}
-      <li>
-        <input type="checkbox" id="notify_new_listed_versions"
-               data-api-url-subscribe-listed="{{ drf_url('reviewers-addon-subscribe-listed', addon.pk) }}"
-               data-api-url-unsubscribe-listed="{{ drf_url('reviewers-addon-unsubscribe-listed', addon.pk) }}"
-               {% if subscribed_listed %}checked="checked"{% endif %} autocomplete="off" />
-          <label for="notify_new_listed_versions">{{ _('Notify me about new listed versions') }}</label>
-      </li>
-      {% if action_allowed(amo.permissions.ADDONS_REVIEW_UNLISTED) %}
-      <li>
-        <input type="checkbox" id="notify_new_unlisted_versions"
-          data-api-url-subscribe-unlisted="{{ drf_url('reviewers-addon-subscribe-unlisted', addon.pk) }}"
-          data-api-url-unsubscribe-unlisted="{{ drf_url('reviewers-addon-unsubscribe-unlisted', addon.pk) }}"
-          {% if subscribed_unlisted %}checked="checked"{% endif %} autocomplete="off" />
-          <label for="notify_new_unlisted_versions">{{ _('Notify me about new unlisted versions') }}</label>
-      </li>
+        {% if acl_is_reviewer or action_allowed(amo.permissions.REVIEWER_TOOLS_VIEW) %}
```

Comment:
can a user who isn't `acl_is_reviewer` even get to this page?

Reply:
Maybe not, but we need some way to limit this part to listed reviewers only, since technically, unlisted and listed reviewers are distinct.